apps/application-system/api/infra/application-system-api.ts

```diff
@@ -137,6 +138,9 @@ export const serviceSetup = (services: {
         staging: 'https://identity-server.staging01.devland.is',
         prod: 'https://innskra.island.is',
       },
+      APPLICATION_SYSTEM_FORM_URL: ref(
```

Comment:
Why is this being added?
This PR only introduce an environmentvariable, but no code change that is using this.  Also why should an restful backend api that is abstracted behind the GQL server need to know the frontend url?